src/event-handler/src/e2e/e2e.test.ts

```diff
@@ -1,70 +1,83 @@
-jest.retryTimes(10)
+jest.setTimeout(50000)
```

Comment:
does this need to be reset?

Reply:
It's a long-running test. I tested it without setting the timeout and it failed. It's because we invoke message receiver lambda 3 times (for 3 events) and we poll S3 for the messages to arrive, then we invoke event handler for each message and poll DynamoDB for the events.